resources/prosody-plugins/mod_auth_jitsi-anonymous.lua

```diff
@@ -57,10 +58,17 @@ end
 module:provides("auth", provider);
 
 local function anonymous(self, message)
+    -- Same as the vanilla anonymous auth plugin
+    local username = generate_random_id():lower();
```

Comment:
Is this uuid?

Reply:
No, it's what the vanilla anonymous auth module uses, which is some combination of letters and numbers.